src/alphafold_impact/pipelines/data_collection_oa/nodes.py

```diff
@@ -0,0 +1,117 @@
+import requests
```

Comment:
Very minor one, but good practice ([C0411](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/wrong-import-order.html)). `math`, `typing`, and `json` should go before `requests` and `tqdm`.